db/schema.rb

```diff
@@ -10,11 +10,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_12_05_233527) do
+ActiveRecord::Schema.define(version: 2018_12_17_175316) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "boards", force: :cascade do |t|
```

Comment:
We probably don't want to commit this where we aren't planning on using this.

Reply:
I do not know where the hell this came from. I promise I didn't even type 'board' yesterday.